elements/element-stuff-base.html

```diff
@@ -37,11 +38,17 @@
             return;
           }
 
+          // FIXME : A bit silly but useful for custom-element cases
+          let _value = target.value;
+          if (!_value) {
+            _value = target.color;
+          }
+
           Polymer.Base.fire('element-updated',
               {
                 type: this.stuffType,
-                name: target.name,
-                value: target.value,
+                name: target.getAttribute('name'),
```

Comment:
why is this change needed?

Reply:
this is linked to the previous issue with the name as an attribute